ambry-store/src/main/java/com.github.ambry.store/PersistentIndex.java

```diff
@@ -1176,18 +1176,23 @@ private void filterDeleteEntries(List<MessageInfo> messageEntries) {
    * @throws StoreException
    */
   void close() throws StoreException {
-    persistor.write();
-    if (hardDeleter != null) {
+    final Timer.Context context = metrics.indexShutdownTimeInMs.time();
```

Comment:
Why do we need `Timer` for these metrics? We don't have to track the rate, do we? A `Histogram` should suffice